graphdatascience/tests/integration/test_model_ops.py

```diff
@@ -164,77 +164,77 @@ def gs_model(runner: Neo4jQueryRunner, gds: GraphDataScience, G: Graph) -> Gener
+    query = "CALL gds.model.drop($name)"
     params = {"name": shared_model.name()}
     runner.run_query(query, params)
 
 
 @pytest.mark.model_store_location
 def test_model_load(runner: Neo4jQueryRunner, gds: GraphDataScience, gs_model: GraphSageModel) -> None:
-    runner.run_query(f"CALL gds.alpha.model.store('{gs_model.name()}')")
-    runner.run_query(f"CALL gds.beta.model.drop('{gs_model.name()}')")
+    runner.run_query(f"CALL gds.model.store('{gs_model.name()}')")
+    runner.run_query(f"CALL gds.model.drop('{gs_model.name()}')")
 
-    model, result = gds.alpha.model.load(gs_model.name())
+    model, result = gds.model.load(gs_model.name())
     assert isinstance(model, GraphSageModel)
     assert result["loadMillis"] >= 0
 
-    runner.run_query(f"CALL gds.beta.model.drop('{gs_model.name()}')")
-    runner.run_query(f"CALL gds.alpha.model.delete('{model.name()}')")
+    runner.run_query(f"CALL gds.model.drop('{gs_model.name()}')")
+    runner.run_query(f"CALL gds.model.delete('{model.name()}')")
 
 
 @pytest.mark.model_store_location
 def test_model_store(runner: Neo4jQueryRunner, gds: GraphDataScience, gs_model: GraphSageModel) -> None:
-    model_name = gds.alpha.model.store(gs_model)["modelName"]
+    model_name = gds.model.store(gs_model)["modelName"]
 
     # Should be deletable now
-    runner.run_query(f"CALL gds.beta.model.drop('{gs_model.name()}')")
-    runner.run_query(f"CALL gds.alpha.model.delete('{model_name}')")
+    runner.run_query(f"CALL gds.model.drop('{gs_model.name()}')")
+    runner.run_query(f"CALL gds.model.delete('{model_name}')")
 
 
 @pytest.mark.model_store_location
 def test_model_delete(runner: Neo4jQueryRunner, gds: GraphDataScience, gs_model: GraphSageModel) -> None:
-    model_name = runner.run_query(f"CALL gds.alpha.model.store('{gs_model.name()}')")["modelName"][0]
+    model_name = runner.run_query(f"CALL gds.model.store('{gs_model.name()}')")["modelName"][0]
 
     model = gds.model.get(model_name)
-    runner.run_query(f"CALL gds.beta.model.drop('{gs_model.name()}')")
-    assert gds.alpha.model.delete(model)["deleteMillis"] >= 0
+    runner.run_query(f"CALL gds.model.drop('{gs_model.name()}')")
+    assert gds.model.delete(model)["deleteMillis"] >= 0
 
-    res = runner.run_query(f"CALL gds.beta.model.exists('{model_name}')")
+    res = runner.run_query(f"CALL gds.model.exists('{model_name}')")
     assert not res["exists"][0]
 
 
 def test_model_drop(gds: GraphDataScience, G: Graph) -> None:
     model, _ = gds.beta.graphSage.train(G, modelName="hello", featureProperties=["age"])
 
-    assert gds.beta.model.exists(model.name())["exists"]
+    assert gds.model.exists(model.name())["exists"]
 
-    assert gds.beta.model.drop(model)["modelInfo"]["modelName"] == model.name()
-    assert not gds.beta.model.exists(model.name())["exists"]
+    assert gds.model.drop(model)["modelInfo"]["modelName"] == model.name()
+    assert not gds.model.exists(model.name())["exists"]
 
 
```

Comment:
I think we should add a test here to make sure the alpha/beta endpoints are still accessible